src/components/Banner/Banner.tsx

```diff
@@ -94,9 +94,7 @@ const variantToIconAssetsMap: {
  * @deprecated The Banner component is deprecated and will be removed in an upcoming release.
```

Comment:
This component comment is currently not appearing at all; will address in a separate PR.